feat(experiments): Manual running time calculator

jurajmajerik opened the pull request:
## Changes
Add a manual calculation mode to the running time configuration modal. Users can now input their own baseline conversion rate (or avg events/property value for count/sum metrics), expected daily exposure  rate, and MDE to calculate sample size and estimated running time. This is useful when planning experiments before launch.

<img width="441" height="669" alt="image" src="https://github.com/user-attachments/assets/6241d9df-e599-4519-a175-c0b5990709eb" />

jurajmajerik committed:
init

jurajmajerik committed:
refactor

andehen reviewed (approved):
Nice, looks good! Tested locally, no bugs seen! Not a blocker, but it would be nice to have some unit tests for these calculations such that we know they are in line with external calculators.

Some nitpicks:
* I'd prefer if the baseline conversion value was set to something typical, ~5 %
* Very minor, but it was a bit annoying that I can't erase the 0 when I want to enter a number

Other thoughts:
* It's hard to do good estimations on count/sum metrics due to the unknown variance. We may need to adjust the assumptions there once we learn a bit more how well the current ones matches real world usage

jurajmajerik committed:
resolve

jurajmajerik committed:
fix input not deleting zero

jurajmajerik commented:
> - I'd prefer if the baseline conversion value was set to something typical, ~5 %
Fixed!
> - Very minor, but it was a bit annoying that I can't erase the 0 when I want to enter a number

Good catch, fixed both!

>It's hard to do good estimations on count/sum metrics due to the unknown variance. We may need to adjust the assumptions there once we learn a bit more how well the current ones matches real world usage

💯
